sway-lsp/src/config.rs

```diff
@@ -20,6 +22,15 @@ pub struct Config {
     pub garbage_collection: GarbageCollectionConfig,
 }
 
+#[derive(Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LspClientConfig {
```

Comment:
how do you feel about just calling this `LspClient` ?

Reply:
I had it that way initially and I can change it if you want. I called it this just because all of the other enums/structs in the Config object have the "Config" prefix.